### PR TITLE
[YUNIKORN-1960] Add updating DOAP file to release tasks

### PR DESCRIPTION
**Pull request by vani-gupta**
### What is this PR for?
A DOAP file is required for all Apache projects. For yunikorn, it is maintained in the yunikorn-site repo: https://github.com/apache/yunikorn-site/blob/master/doap_YuniKorn.rdf. This file should be updated as part of uploading the release notes to the website. This PR adds that step to the release procedure. 

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1960

### How should this be tested?
Th modified .md file can be rendered by any supported tool to view the changes. 

### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/pages/community/release_procedure.md [hunk 1]
```diff
@@ -61,9 +61,10 @@ Simplified release procedure:
 2. Stabilize the release by fixing test failures and bugs only
 3. Tag update release for a new version to prepare a release candidate, e.g `v1.3.0-1` for RC1
 4. Update the CHANGELOG
-5. Configure [release-configs.json](https://github.com/apache/yunikorn-release/tree/master/tools/release-configs.json)
-6. Run script [build-release.py](https://github.com/apache/yunikorn-release/tree/master/tools/build-release.py) to generate source code tarball, checksum and signature.
-7. Voting and releasing the candidate
+5. Update the DOAP file
```
Comment (wilfred-s) [inline src/pages/community/release_procedure.md:64]: The release date is not known until we have voted and approved the release.

**Review by wilfred-s — CHANGES_REQUESTED** [review]
The doap file should be updated as part of the website update, we do not know the release date until we have voted and approved everything.
The text is correct, just move the whole heading to the section: `Website updates for a new release`. No need to add an item.

File: src/pages/community/release_procedure.md (hunk 1 above)
Comment (vani-gupta) [reply src/pages/community/release_procedure.md]: Added this section under `Website updates for a new release` as suggested.

**Review by wilfred-s — APPROVED** [review]
LGTM

**Comment by wilfred-s** [comment]
Waiting for the build to finish, if it passes I'll commit the change

**Comment by vani-gupta** [comment]
Thanks for the review, @wilfred-s

**Pull request closed without merging**